scss/detail.scss

```diff
@@ -0,0 +1,48 @@
+@import 'variables';
+
+.text{
+  font-family: 'Roboto', sans-serif;
+}
+
+.container-text{
+  display: flex;
+  flex-direction: column;
+}
+
+.container-secondary-text{
+  display: flex;
+  flex-direction: column;
+}
+
+.book-title{
```

Comment:
Since the elements that use the classes `.book-title` and `.title-genre` are nested inside it's parent in the html we can do the same in the scss. Thanks to sass nesting (look for the "nesting" headingin this [guide](https://sass-lang.com/guide)) we can nest classes to add specifity and have more control over styles and even reduce the name of the classes

For example:
```scss
.container-title {
  .title {
    ...
  }
  
  .genre {
    ...
  }
}
```

So in this case the app will know that every element with the class "title" inside an element  with the class ".container-title" will have an specific styling

Reply:
So, to use it, it would be something like  "class = containter-title.genre"  ???

Reply:
No, the HTML layout can remain as it is (keep in mind that if ypu change a class name you will have to change that name in the HTML)